### DISCUSSION TITLE
Amp scoring action

### DISCUSSION THREAD

**Discussion by PChild** [team401/2024-Robot-Code#38] [Ideas]
Should amp scoring mode go in #30 ? The current plan has a Vortex driving a gear to move a hood into a specific position and redirect shots out of the shooter. The shooter would go to a low power spit speed, the hood would move to position, the arm would move to a specific position to score out the same side as the intake, and then the indexer would feed the note into the shooter wheels.

**Comment by jkleiber** [comment]
Let's make a new issue in the scoring mvp milestone so we can describe this behavior. It does stand alone enough imo
